CMSgov/mint-app
# Easi 1859 job codes and roles

# EASI-1859

## Changes and Description

- Removed references to GRT and 508 job codes and roles
- Added job code/roles for a basic user and admin user

I lieu of a job code, basic role is applied to user
Local login auth only
No dev job code for basic - same as in EASI

## How to test this change

![Screen Shot 2022-03-30 at 3 46 47 PM](https://user-images.githubusercontent.com/95709965/160918396-8f9bc0bd-59c5-4f8e-b7f6-d3da41587359.png)

Log in with each job code from the local auth login
Navigate to `http://localhost:3005/user-diagnostics` and verify the user's job code and role

![Screen Shot 2022-03-30 at 3 49 16 PM](https://user-images.githubusercontent.com/95709965/160918824-e7c24a13-ef74-4a2c-82ac-e8d88eef925d.png)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
